apps/src/StudioApp.js

```diff
@@ -1,5 +1,14 @@
-// Globals:
-//   Blockly
+/* jshint
+  strict: true,
+  curly: true,
```

Comment:
This option requires you to always put curly braces around blocks in loops and conditionals.